### DISCUSSION TITLE
LCD 16x2: Is there any example about display information from the DAW?

### DISCUSSION THREAD

**Discussion by Vahen1981** [tttapa/Control-Surface#444] [Q&A] [answered]
Hello everyone!
My project for a control surface is almost done, but I would like to show some few information extracted from the daw in my lcd 16x2 with IC2 protocol.
Could someone write a little example about how to do it?? Just something like, display the time of the daw, or the name of a track...
I tried to do some changes in the oled examples, but I'm really lost on this and I didn't get nothing... So if someone could show me something specifically for an lcd 16x2 I'll be greatfull
Thanks!

**Comment by tttapa** [comment, marked as answer]
Try to get the LCD display working with simple static text first (without MIDI or Control Surface). 

Most Control Surface classes have methods to get access to the state or text of the MIDI elements, for example, [`LCD::getText()`](https://tttapa.github.io/Control-Surface-doc/new-input/Doxygen/d5/d2e/classMCU_1_1LCD.html#af047a13ee1013ac5e28dd550a9ddb0c1).
